TopsortAnalytics/build.gradle

```diff
@@ -41,13 +41,18 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.9.0'
 
+    //Gson
+    implementation 'com.google.code.gson:gson:2.9.0'
+
```

Comment:
Why did you add gson back?

Reply:
Just comparing results, and it won't compile/test without it while I didn't chance the Cache class
It will be removed 🔪